### DISCUSSION TITLE
how do I make the website work agen plese

### DISCUSSION THREAD

**Discussion by ghost** [TeamNewPipe/NewPipe#11699] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

![image](https://github.com/user-attachments/assets/738823ab-d93e-411b-bd11-2e85630d2f27)
this

### Additional information

I wont to know why it is not working plese

**Comment by AudricV** [comment, marked as answer]
Piped is not NewPipe. Report Piped issues at the corresponding issue trackers (this one is already known and affects both clients) and look at existing issues to understand why it doesn't work.
